tests/acceptance/TestHelpers/GraphHelper.php

```diff
@@ -16,6 +16,21 @@
  * A helper class for managing Graph API requests
  */
 class GraphHelper {
+	public static array $DEFAULT_PERMISSIONS_ROLES = [
```

Comment:
let's make it a constant varaible
```suggestion
	const DEFAULT_PERMISSIONS_ROLES = [
```